docs/pages/eas-update/rollouts.mdx

```diff
@@ -1,32 +1,76 @@
 ---
 title: Rollouts
-description: Incrementally deploy updates on a new branch to a channel.
+description: Incrementally deploy updates to your users.
 ---
 
+import { Collapsible } from '~/ui/components/Collapsible';
 import { Terminal } from '~/ui/components/Snippet';
 
-> **info** Available from SDK 50 using [`expo-updates`](/versions/latest/sdk/updates/) 0.23.0 or above.
+A rollout allows you to roll out a change to a portion of your users in order to catch bugs or other issues before releasing that change to all your users.
 
-Rollouts incrementally deploy updates on a new branch to a specific channel. With rollouts, you can roll out updates from one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to our production environment.
+EAS provides two rollout mechanisms depending on your use case:
 
-## Start a rollout
+## Per-update rollouts
 
-To start a rollout, ensure you have EAS CLI version `4.0.0` or above installed. Then, run the following command:
+This rollout mechanism allows you to specify a percentage of users that should recieve a new update when you publish it, and then increase that percentage gradually afterwards.
+
+<Collapsible summary="How to use">
```

Comment:
Not sure why we need collapsibles in both sections. IMO this makes the doc difficult to scan since as the reader, I am already seeing the instructions in the ToC:

![CleanShot 2024-10-22 at 01 35 01@2x](https://github.com/user-attachments/assets/c5ff89d9-da19-4ed5-8606-019412a696ef)

This pattern doesn't allow me to quickly scan the document and adds an extra click to uncollapse. Also, removing the collapsible would eliminate the need to explain "how to use" each mechanism.